web/src/components/application-detail-page/application-detail/health-status-icon/index.tsx

```diff
@@ -31,6 +34,8 @@ export const ApplicationHealthStatusIcon: FC<ApplicationHealthStatusIconProps> =
         return <FavoriteIcon className={classes.healthy} />;
       case ApplicationLiveStateSnapshot.Status.OTHER:
         return <OtherIcon className={classes.other} />;
+      case ApplicationLiveStateSnapshot.Status.UNHEALTHY:
+        return <OtherIcon className={classes.unhealthy} />;
```

Comment:
Use OtherIcon similar to `OTHER`'s one  as the `UNHEALTHY` icon for now.

This is a fix for lint/web.

Reply:
@t-kikuc 
Thanks, I agree with it.
Fixed at [b723484](https://github.com/pipe-cd/pipecd/pull/5454/commits/b7234846658801220a2a8024ae27160a6a651f3f)